airbyte-integrations/bases/debezium-v1-9-2/src/main/java/io/airbyte/integrations/debezium/internals/PostgresConverter.java

```diff
@@ -84,11 +138,7 @@ private void registerMoney(final RelationalColumn field, final ConverterRegistra
         return DebeziumConverterUtils.convertDefaultValue(field);
       } else if (x instanceof Double) {
         final BigDecimal result = BigDecimal.valueOf((Double) x);
-        if (result.compareTo(new BigDecimal("999999999999999")) == 1
```

Comment:
is this no longer needed? I saw some comments in the deleted code about money values being rounded (somehow related to https://github.com/airbytehq/airbyte/issues/7338)

Reply:
Yes, you're right. Currently, this code is needed until #7338 will be fixed, so I reverted it. I tried to fix that in the scope of this ticket, but it requires more time than I expected